### PR TITLE
[docs] add LTS and STS policies

### PR DESCRIPTION
**Pull request by polarweasel**
Getting the LTS and STS policies in place.

@netlify /preview/releases/

### CODE REVIEW

**Review by tylarb — DISMISSED** [review]
We'll need to add a section for the YBA release schedule as well - this isn't covered yet

File: docs/content/preview/releases/_index.md [hunk 1]
```diff
@@ -15,31 +15,36 @@ menu:
...
 ### Release support policy
 
-Support for YugabyteDB stable release series includes:
+The type of YugabyteDB release series you are running determines its support timelines.
+
+**LTS (long-term support)** release series receive maintenance updates for at least 18 months (545 days) from the first release date of the minor release.
+
+**STS (short-term support)** release series receive maintenance updates for at least 8 months (240 days) from the first release date of the minor release.
+
+**Stable** release series receive maintenance updates for at least 12 months (365 days) from the first release date of the minor release.
```
[the remark this answers is not in the document]
Comment (polarweasel) [reply docs/content/preview/releases/_index.md]: I've adjusted this now

**Comment by polarweasel** [comment]
> We'll need to add a section for the YBA release schedule as well - this isn't covered yet

@tylarb We can do that when the time comes. For now, I've adjusted some wording to mention "YugabyteDB releases" specifically, rather than Yugabyte releases. That'll make it easier to add YBA.

File: docs/content/preview/releases/_index.md [hunk 2]
```diff
@@ -15,31 +15,36 @@ menu:
...
+For details about the differences between the release series, see [Release versioning](versioning/).
 
 ### Release support policy
 
-Support for YugabyteDB stable release series includes:
+The type of YugabyteDB release series you are running determines its support timelines.
+
+**LTS (long-term support)** release series receive maintenance updates for at least 18 months (545 days) from the first release date of the minor release.
+
+**STS (short-term support)** release series receive maintenance updates for at least 8 months (240 days) from the first release date of the minor release.
```
[the remark this answers is not in the document]
Comment (polarweasel) [reply docs/content/preview/releases/_index.md]: Done

**Pull request merged**